web/forms.py

```
@@ -1869,3 +1873,53 @@ class StudyGroupForm(forms.ModelForm):
+            ),
+            "category": TailwindSelect(
+                attrs={
+                    "class": (
+                        "w-full px-4 py-2 border border-gray-300 dark:border-gray-600"
+                        " rounded-lg focus:ring-2 focus:ring-blue-500",
+                    )
+                }
+            ),
+        }
+
+    def __init__(self, *args, **kwargs) -> None:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)